data/tutorials/language/0it_00_values_functions.md

```diff
@@ -209,15 +214,103 @@ val forename : string = "Robin"
 val surname : string = "Milner"
 ```
 
+### Nested Pattern Matching on User-Defined Types
+
+Pattern matching also works with nested user-defined types.
+
+In the example below, we deconstruct nested tuples:
+```ocaml
```

Comment:
Generally it's a good idea to leave a blank line between code blocks and the surrounding text.

Reply:
(even if the code in the guides doesn't do this consistently)